### PR TITLE
Big bug

### LINKED ISSUE #176: fix popup that shows if customer has ordered too much.
popup disappears informing customer has ordered more than 20, need to make it so that it displays the whole time.

### LINKED ISSUE #180: Implement textbox attached to each order in active order list
This textbox will be used to jot notes on why you are accepting or denying an order, will be sent to customer email not database.

### LINKED ISSUE #222: Integrate Location Selection into order history

### LINKED ISSUE #234: Alert for missing location field
Clearly notify user the required field must be filled (location)

### LINKED ISSUE #260: separate css into different files
- filing stuff in general

### LINKED ISSUE #283: Make fire animation state; depend on hover

### LINKED ISSUE #287: Remove nodeMailer

### PR DESCRIPTION
**Pull request by SpeiserK**
- Remove nodemailer
- re-fix amp banner
- Phone number format for payment summary and customer order history

close #287 
close #222 
close #180 
close #283 
close #260 
close #234 
close #176

### CODE REVIEW

File: MERN Scripts/backend/routes/OrderRoute.js
Comment (zachpren0) [inline MERN Scripts/backend/routes/OrderRoute.js:1]: Its morphin time

File: MERN Scripts/mern-emsystem/src/components/Empbanner.js
Comment (zachpren0) [inline MERN Scripts/mern-emsystem/src/components/Empbanner.js:1]: love it

File: MERN Scripts/mern-emsystem/src/components/PaymentSummary.js
Comment (zachpren0) [inline MERN Scripts/mern-emsystem/src/components/PaymentSummary.js:1]: solving problem

File: MERN Scripts/mern-emsystem/src/pages/CustOrderHist.js
Comment (zachpren0) [inline MERN Scripts/mern-emsystem/src/pages/CustOrderHist.js:1]: yes

**Review by zachpren0 — APPROVED** [review]
Great pr

**Review by amurph24 — APPROVED** [review]
Big bug 
image

**Review by camwlsn — APPROVED** [review]
Nice work on fixing the formatting of phone numbers across all pages. 😃

**Pull request merged**